### DISCUSSION TITLE
posts/github-graph-beds-cdn/

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#9] [Announcements]
# posts/github-graph-beds-cdn/

通过 Cloudflare 和 jsDelivr 免费加速博客 GitHub 静态资源(GitHub图床)，自动实现 CDN 资源的海内外分流，加速博客访问速度。

https://www.haoyep.com/posts/github-graph-beds-cdn/

**Comment by huxxhua** [comment]
大佬，不知道是不是我操作的问题，用 Chinajsdelivr 进行加速吗，按教程用jsd.cdn.zzko.cn，返回404，图片无法加载。

另外看到在国内访问您的图片跳转的是 Vercel 服务的img域名
有请大佬指导  ==

**Reply by leegical** [reply]
Chinajsdelivr的审核可能比较严，把我的仓库封了。目前博客用的是vercel加速。以后有空写个教程吧

**Reply by leegical** [reply]
更新了，使用vercel加速Github图床
https://www.haoyep.com/posts/github-graph-beds-vercel-cdn/

**Comment by callmeyellowbin** [comment]
大佬，看到您今天更新了文章，我发现我之前图床上所有的图片都Not Found了，是不是修改了什么，要怎么解决呢，谢谢大佬！

**Reply by leegical** [reply]
如果你使用的是Chinajsdelivr，可以去 jsd.cdn.zzko.cn 看一下是不是自己的图床仓库被封了

**Comment by miaoyanginfo** [comment]
大佬我按你的教程部署了，现在的问题是访问cdn.我的域名的时候http会自动跳转到https，提示站点不安全，无法访问，请问该如何处理处理呢？

**Reply by miaoyanginfo** [reply]
还有一个问题，我在浏览器访问cdn.我的域名之后会自动跳转到规则设置的源网址，导致在文章页面无法显示图片。

**Reply by miaoyanginfo** [reply]
http://cdn.miaoyang.win/gh/miaoyanginfo/hexo-img/img/23ce233a6778940eb9998f5f.jpg

**Reply by leegical** [reply]
如果按照我的配置，出来的链接应该就是https，你再看看是不是哪里没有配置好

**Reply by miaoyanginfo** [reply]
之前没有配置SSL证书，订阅saas申请证书之后正常了。感谢
